pkgs/applications/misc/fnott/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, lib
+, fetchgit
+, pkg-config
+, meson
+, ninja
+, scdoc
+, wayland-protocols
+, tllist
+, fontconfig
+, freetype
+, pixman
+, libpng
+, wayland
+, wlroots
+, dbus
+, fcft
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fnott";
+  version = "1.1.0";
+
+  src = fetchgit {
```

Comment:
this should be fetchFromGitea.

Reply:
thanks, I didn't know fetchFromGitea existed. I will open another PR.